src/openrct2/actions/TrackPlaceAction.cpp

```diff
@@ -104,6 +104,13 @@ GameActions::Result TrackPlaceAction::Query() const
             GameActions::Status::InvalidParameters, STR_RIDE_CONSTRUCTION_CANT_CONSTRUCT_THIS_HERE, STR_ERR_VALUE_OUT_OF_RANGE);
     }
 
+    if (_brakeSpeed > kMaximumBrakeSpeed)
+    {
+        LOG_WARNING("Invalid speed for track placement, speed = %d", _brakeSpeed);
+        return GameActions::Result(
+            GameActions::Status::InvalidParameters, STR_RIDE_CONSTRUCTION_CANT_CONSTRUCT_THIS_HERE, STR_SPEED_TOO_HIGH);
```

Comment:
The value checked is `break speed` and I can build a special track type where I can select `break speed`. I would expect the error message to use the same nomenclature - currently it only mentions `speed`

Reply:
Speed is also used for boosters, and this error message is only expected to pop up when plugins send the wrong data to the game action (because the ride construction window will simply not send a game action if the speed is already at its maximum). As such, it seemed a bit of a waste to have two separate strings. Nevertheless, I can still change that if you wish.

Reply:
OK